building-and-research/human-in-the-loop.md

```diff
@@ -0,0 +1,45 @@
+---
+nav_order: tbd
+parent: Building and research
+title: Human-in-the-loop
+description: Machine translation supported by human post-editing or evaluation
+---
+
+## Definition
+
+Human-in-the-loop consists of using human feedback for additional training of translation engines. Feedback is provided in one of the following ways:  
+* Humans correct post-edited machine translation
+* Humans annotate errors in machine translation
+  
+Other human-machine interactions are also considered human-in-the-loop:
+* Humans improveme source content for better translatability
+* Humans label training data to classify various domains or quality levels
+* Fallback option to human translation in case an automated solution is inadequate
+
+## Goal
+
+The goal of human-in-the-loop is improving the quality of machine translation output in all aspects:
+* Accuracy - eliminating factual errors and hallucinating
+* Fluency - making the language sound more natural for native speakers
+* Terminology and style - using appropriate terms and style in given context
+
+## Tasks
+
+These tasks can be performed by individuals or teams, depending on the specific requirements of the process that involves the machine translation and on the expected main purpose of translated content.
+
+1. Pre-editing: Before feeding text into the machine translation system, human editors may pre-edit the source text to ensure that it is clear, concise, and easy to translate.
+
+2. Machine Translation: The pre-edited text is then fed into a machine translation system, which generates an initial translation.
+
+3. Post-editing: A human post-editor reviews the machine-generated translation and makes any necessary corrections to improve the quality of the final output. This task is performed by professional translators.
+
+4. Quality Assessment: Human quality assessors may evaluate the quality of the machine-generated translations and provide feedback to the machine translation system to help it improve over time. This task is performed by persons equipped with a checklist and list of error weights.
```

Comment:
Maybe the last sentence can be edited to omit the actor of the task, since it's too vague.

---
Checklists and error weights are relevant tools for quality assessment.

Reply:
I edited the last sentence to "This task requires a checklist of error categories and weights."